### ISSUE TITLE
Identity > Typography page > Web type accessibility section

### ISSUE THREAD

**Issue by iuloshi** [cfpb/design-manual#100]
In the "More information" section, what are the URLs the bracketed words are supposed link to?

![screen shot 2014-04-01 at 1 44 54 pm](https://cloud.githubusercontent.com/assets/1676307/2582523/0266a906-b9c6-11e3-8e2d-87173b2a2395.png)

**Comment by iuloshi** [comment]
This guy perhaps? (Thanks AJ!) http://accessibility.oit.ncsu.edu/tools/color-contrast/

**Comment by Scotchester** [comment]
Yeah that's the one for `[This color contrast tool]`. The **Important note** is a weird remnant from the DPG that should be deleted entirely.

**Comment by iuloshi** [comment]
Got it, I'll update.
